### PR TITLE
Support for unusual characters on a view's name

### PR DESCRIPTION
**Pull request by aokiji**
I have to support latin1 characters on some view's name, so I made a few fixes so that I could still use scenic.

### CODE REVIEW

File: lib/scenic/adapters/postgres/views.rb [hunk 1]
```diff
@@ -54,6 +54,13 @@ def to_scenic_view(result)
             materialized: result["kind"] == "m",
           )
         end
+
+        private
+
+        def pg_identifier(name)
+          return name if name =~ /^[a-zA-Z_][a-zA-Z0-9_]*$/
+          PGconn.quote_ident(name)
```
Comment (derekprior) [inline lib/scenic/adapters/postgres/views.rb]: Can we always quote the name? Or will that not work?

File: spec/scenic/schema_dumper_spec.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 require "spec_helper"
 
 class Search < ActiveRecord::Base; end
+class SearchInAHaystack < ActiveRecord::Base; self.table_name = '"search in a haystack"'; end
```
Comment (calebthompson) [inline spec/scenic/schema_dumper_spec.rb]: This can be split to three lines.

**Comment by calebthompson** [comment]
@aokiji I deleted a few pieces of Hound feedback I'm comfortable ignoring here, but if you don't mind taking a look at the rest we can try and get this out in the upcoming release. It's mostly whitespace changes and splitting a few things over multiple lines.

**Comment by calebthompson** [comment]
If you rebase onto master, it may clear up some of the CI issues as well.

**Comment by aokiji** [comment]
I rebased and acted on some of the feedback from Hound. Now all tests are passing. @calebthompson I would be very glad to see this change on the next release

**Comment by calebthompson** [comment]
Merged in #188

**Pull request closed without merging**